src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java

```diff
@@ -46,8 +46,8 @@ public ComponentVersionPolicyViolationCount(final NameValuePairView nameValuePai
             name = PolicySeverityType.valueOf(nameValuePair.name);
         }
 
```

Comment:
Not in love with this, but hub-common-api has (correctly) changed value from String to Object, and Gson converts it to a Double. Any other ideas?

Reply:
Are you against something like this?
` if (nameValuePair.value != null) {
            value = ((Double) nameValuePair.value).intValue();
         }`

Reply:
If your worried about the casting, them I like your approach for now